restapi/src/main/java/io/stargate/web/resources/Converters.java

```diff
@@ -72,6 +74,14 @@
 public class Converters {
 
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+  static {
+    // For Java 8 date/time types we need:
+    OBJECT_MAPPER.registerModule(new JavaTimeModule());
```

Comment:
Without this `java.time.Instant` becomes that odd JSON Object which can not be deserialized (nor easily used from Java). Module adds explicit serializer, deserializer.